### ISSUE TITLE
Support for latest stable version of Newtonsoft.Json (7.x.x.x)

### ISSUE THREAD

**Issue by gthuruv** [aspnet/dnx#3136]
Default dnx451 has dependency on Newtonsoft.Json 6.0.6. Newtonsoft's latest stable version is 7.0.1. Is it possible to update the reference to the latest version of the package?

**Comment by Eilon** [comment]
You can use any version of Newtonsoft.Json 6.0.6 or higher - just specify it in your `project.json` file.

**Comment by gthuruv** [comment]
Thanks!

**Comment by tebeco** [comment]
I'm not sure how to do that because this is what VS2015 looks like when I try with RC1-Final :
It shows error for all aspnet packages that depends on Newtonsoft 6.0.x (Almost 90%)

What's the proper way to deal with it ?
WebSite (xproj A : bottom project) reference a class library (xproj B : dnx package type) and this one reference a third project (xproj C : dnx package type)

A > B > C
* A is a Web site template for MVC6 (aspnet Preview)
* 'C' depends on Newtonsoft.Json : "NewtonSoft.Json": "8.0.1-beta3",
* 'B' depends on 'C' : "MyProduct.C": "1.0.0-*",
* 'B' have the package "Microsoft.AspNet.Server.Kestrel": "1.0.0-rc1-final" (that depends on NewtonSoft 6.0.x
* 'A' depends on B
* 'A' also depends on about 10 packages that depends directly or not on Newtonsoft 6.0.x

It completly breaks Packages restoration for both ".NetFramework 4.5.1" and "Dnx 5.4"

How should I handle this issue ?

![bugnewtonsoftjson](https://cloud.githubusercontent.com/assets/2266487/11362284/f68ae1c2-9292-11e5-90b6-91e2a406ce9f.png)

**Comment by Eilon** [comment]
@anurse @davidfowl ?

**Comment by anurse** [comment]
Can you run `dnu restore` for the projects that are showing the error on the command line and post the output?

**Comment by tebeco** [comment]
I will in about 10 hours ;)

**Comment by tebeco** [comment]
Hi again,
FYI : I'm trying to migrate libraries from .net 2/4 to Dnx Compliant package.

Here is here I am :
* I've removed Newtonsoft.Json from all three project.json
* I've still found evidence of it inside Lock file
* I've cleaned project.lock.json (it was corrupted because of the error, i'm guessing there's a bug un VS2015 when you add/remove dependencies sometimes it does NOT CLEAN the lock files, it happens more that 5 times already on different occasions)
* I've removed ALL the code that was requiring Newtonsoft from project 'C'
* I've Added again Newtonsoft.Json inside Project 'C'
* I've added Code that depends on newtonsoft (just a POC) in project 'C'

So far i can't managed to reproduce the issues I had.
Believe it or not i really think the is indirectly caused by Corporate Proxy i'm using here.
I was using a "Guest" Wifi (no proxy), I thought it as working until this specific issue.
I had to go back to the Proxy Network so I edited the "Machine.Config" for v4.xxx framwork/framework64 with the common block : 
      <proxy usesystemdefault="True"/>

And it seems that this AND the clean up fixed few issues
Sorry about that :(

Question aside :
Is there any extension planned inside Visual Studio to have a more user friendly UI ou Test connectivy, dnvm version, dnx ... like the Cloud Explorer for example that gives a side view but could be managed aside from the Dev of VS2015 it self
